apps/api/src/app/bridge/bridge.controller.ts

```diff
@@ -113,6 +113,7 @@ export class BridgeController {
         workflows: body.workflows,
         bridgeUrl: body.bridgeUrl,
         source,
+        origin: body.origin || WorkflowOriginEnum.EXTERNAL,
```

Comment:
check if we should move this default to the dto interface instead of having it here.